src/encoded/static/components/browse/components/above-table-controls/SelectedFilesDownloadButton.js

```diff
@@ -107,7 +108,11 @@ export class SelectedFilesDownloadButton extends React.PureComponent {
     }
 
     render(){
-        const { selectedFiles, filenamePrefix, children, disabled, windowWidth, context, ...btnProps } = this.props;
+        const {
+            selectedFiles, filenamePrefix, children, disabled,
+            windowWidth, context, analyticsAddFilesToCart,
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['windowWidth' is assigned a value but never used.](https://app.codacy.com/manual/4dn/fourfront/pullRequest?prid=4885443)